config/samples/operator_v1beta1_compassmanagermapping.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: operator.kyma-project.io/v1beta1
+kind: CompassManagerMapping
+metadata:
+  labels:
+    app.kubernetes.io/name: compassmanagermapping
+    app.kubernetes.io/instance: compassmanagermapping-sample
+    app.kubernetes.io/part-of: compass-manager
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/created-by: compass-manager
+  name: compassmanagermapping-sample
+spec:
```

Comment:
I think we could have missed one thing here. As noted [in the issue concerning resource watching](https://github.com/kyma-project/compass-manager/issues/9#issuecomment-1578657762) we should consider setting statuses for operations. As we can't set status on the Kyma resource, `CompassManagerMapping` seems to be the right place.  

Why this is needed? 

Let's imagine situation when we have Compass outage. It will we surely visible in the logs, as Compass Manager will constantly retry requests.  If we had proper alerting/monitoring we could also be notified that some problem occurred. 
I see the following benefits of adding the status property:
- We would get additional insight into the system:
  - We could see how many runtimes that have module enabled cannot be registered due to Compass failures.
  - We could see when Compass Manager noticed module is enabled, and when runtime was registered, and secret with configuration created
- We could add some dashboards 

You can see the example of status structures [here](https://github.com/kyma-project/infrastructure-manager/blob/0d22ae07324e71670cd43b37ecd4c88d3a4b4d21/api/v1/gardenercluster_types.go#L77).

I think this needs some discussions before we decide to implement it. If we decide to go with statuses let's do that in the separate PR.